sdk/storage/Azure.Storage.Blobs.Batch/tests/BlobBatchClientTests.cs

```diff
@@ -573,6 +573,7 @@ public async Task Batch_Double_Dispose_Response_Still_Available()
         }
 
         [Test]
+        [Ignore("service bug")]
```

Comment:
Nit - let's file a bug in our repo to re-enable this

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/13507